.github/workflows/rust.yml

```diff
@@ -44,3 +44,12 @@ jobs:
 
       - name: Test CLI
         run: make test-cli
+
+      - name: Build resolc
+        run: make install-bin
+
```

Comment:
This should not be required? Since the test-* targets do it anyways

Reply:
Right, fixed